frontend/components/MainPage/header/AgentButton/AgentRunningButton.tsx

```diff
@@ -76,7 +75,8 @@ export const AgentRunningButton = () => {
             Agent is working
           </Paragraph>
         )}
-        <LastTransaction />
+
+        <LastTransaction serviceConfigId={serviceConfigId} />
```

Comment:
Fine with either approach for now, we can optimize at later point!

My preference atm is to use hooks to isolate  children from parent changes where we can, though this may eventually lead to some performance issues. Yet to hit any so far however, but will memoize components/props once the time comes!